lib/kube/proxy/forwarder.go

```diff
@@ -493,6 +507,42 @@ func (f *Forwarder) setupContext(ctx auth.Context, req *http.Request, isRemoteUs
 	return authCtx, nil
 }
 
+func (f *Forwarder) authorize(ctx context.Context, actx *authContext) error {
+	if actx.teleportCluster.isRemote {
```

Comment:
I'd explain via a comment why authz is skipped for remote clusters for the readers. Also, think it might be worth to log this at the debug level?

Reply:
done